Dockerfile-dev

```diff
@@ -0,0 +1,80 @@
+FROM elixir:1.7.3-slim as builder
+
+# Before running following commands, configure
+# 1. 'env/local.env' file.
+# 2. `/apps/snitch_core/config/dev.exs' file
+
+EXPOSE 4000
+
+ENV APP_HOME /avia-backend
+ENV AWS_ACCESS_KEY_ID dasdas
+ENV AWS_SECRET_ACCESS_KEY dasdas
+ENV BUCKET_NAME dsadas
+ENV AWS_REGION dasda
+ENV SENDGRID_API_KEY dassad
+ENV PASSWORD_RESET_SALT dsadas
+ENV TOKEN_MAXIMUM_AGE sdasda
+ENV FRONTEND_CHECKOUT_URL http://localhost:4200/checkout/
+ENV HOSTED_PAYMENT_URL http://localhost:3000/api/v1/hosted-payment/
+ENV SUPPORT_URL https://admin.aviacommerce.org
+ENV WKHTML_PATH /usr/bin/wkhtmltopdf
+ENV ELASTIC_HOST http://localhost:9200/
+
+RUN mkdir $APP_HOME
+WORKDIR $APP_HOME
+
+ADD . $APP_HOME
+
+#Install git
+RUN apt-get update \
+    && apt-get install -y git
+
+# Install essential packages for application build
+RUN apt-get clean \
+  && apt-get update \
+  && apt-get install -y apt-utils apt-transport-https curl git make inotify-tools gnupg g++ \
+  && curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+  && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
+  && curl -sL https://deb.nodesource.com/setup_8.x | bash \
+  && apt-get install -y nodejs yarn
+
+# Umbrella
+COPY mix.exs mix.lock ./
+COPY config config
+
+# Apps
+COPY apps apps
+RUN mix do local.hex --force, local.rebar --force
+RUN mix do deps.get, deps.compile
+
+# Create assets build for admin app
+RUN cd apps/admin_app/assets \
+  && yarn install \
+  && yarn deploy \
```

Comment:
@vikramjadon921995 
```
yarn deploy \
&& cd .. \
&& mix phx.digest
```

not required, as these are for prod buil

Reply:
Okay. Will remove them.